dubbo-common/src/main/java/org/apache/dubbo/common/utils/ObjectUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.dubbo.common.utils;
```

Comment:
Please add ASF license header

Reply:
OK, I'll also change the other documents

Reply:
> Please add ASF license header

done